toolchain/driver/testdata/error_codegen_objcode_target_triple.carbon

```diff
@@ -0,0 +1,8 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// RUN: %{not} %{carbon} dump assembly --target_triple=x86_687-unknown-linux-gnu %s | %{FileCheck-strict}
+// CHECK:STDERR: ERROR: {{.*}}
```

Comment:
I would like to see at least some of the error message here, so that we can have confidence that this test is testing the right thing. You should include something in the error message about the `--target_triple=` flag being invalid; that way users will know what they got wrong (that won't be in the LLVM error message) and also we can avoid test failures if the LLVM error string changes by keeping that part of the diagnostic inside `{{.*}}`.